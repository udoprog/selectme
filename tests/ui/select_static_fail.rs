pub fn main() {
}

/// Regular `select!` does not support that `static` option.
async fn error_default_static() {
    selectme::select! { 
        static;
    };
}

/// Static can only be specified once.
async fn error_multiple_static() {
    selectme::inline! { 
        static;
        static;
    };
}
