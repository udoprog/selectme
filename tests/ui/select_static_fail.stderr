error: `static` option is only supported with the `inline!` macro
 --> tests/ui/select_static_fail.rs:7:9
  |
7 |         static;
  |         ^^^^^^

error: `static` option may only be specified once
  --> tests/ui/select_static_fail.rs:15:9
   |
15 |         static;
   |         ^^^^^^

error: `static` option previously specified here
  --> tests/ui/select_static_fail.rs:14:9
   |
14 |         static;
   |         ^^^^^^
